studio/components/to-be-cleaned/Dropdown/OrganizationDropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from 'next/router'
```

Comment:
Can this file be moved out of the `to-be-cleaned` folder? Or does this need to wait until _all_ the dropdowns are migrated?

Reply:
Maybe leave this, it will be replaced with changes in another PR anyway from @joshenlim